docs/installation.rst

```diff
@@ -131,28 +127,22 @@ This will pull in the dependencies and activate the git head as the current
 version inside the virtualenv.  Then all you have to do is run ``git pull
 origin`` to update to the latest version.
 
-
 .. _windows-easy-install:
 
 `pip` and `setuptools` on Windows
 ---------------------------------
 
 Sometimes getting the standard "Python packaging tools" like *pip*, *setuptools*
-and *virtualenv* can be a little trickier, but nothing very hard. The two crucial
-packages you will need are setuptools and pip - these will let you install
-anything else (like virtualenv). Fortunately there are two "bootstrap scripts"
-you can run to install either.
+and *virtualenv* can be a little trickier, but nothing very hard. The crucial
+package you will need is pip - this will let you install
+anything else (like virtualenv). Fortunately there is a "bootstrap script"
+you can run to install.
 
-If you don't currently have either, then `get-pip.py` will install both for you
-(you won't need to run ez_setup.py).
+If you don't currently have *pip*, then `get-pip.py` will install it for you.
 
 `get-pip.py`_
 
-To install the latest setuptools, you can use its bootstrap file:
-
-`ez_setup.py`_
-
-Either should be double-clickable once you download them. If you already have pip,
+It should be double-clickable once you download it. If you already have pip,
```

Comment:
I think (but not 100% sure) that pip should be changed to code formatting here too